src/server/pegasus_server_impl.cpp

```diff
@@ -293,7 +293,43 @@ pegasus_server_impl::pegasus_server_impl(dsn::replication::replica *r)
     bool disable_bloom_filter = dsn_config_get_value_bool(
         "pegasus.server", "rocksdb_disable_bloom_filter", false, "Whether to disable bloom filter");
     if (!disable_bloom_filter) {
-        tbl_opts.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10, false));
+        // average bits allocated per key in bloom filter.
+        // bits_per_key    |           false positive rate
+        //                 | format_version < 5 | format_version = 5
+        //       6                5.70953              5.69888
+        //       8                2.45766              2.29709
+        //      10                1.13977              0.959254
+        //      12                0.662498             0.411593
+        //      16                0.353023             0.0873754
+        //      24                0.261552             0.0060971
+        //      50                0.225453             ~0.00003
+        // Recommend using no more than three decimal digits after the decimal point, as in 6.667.
+        // More details: https://github.com/facebook/rocksdb/wiki/RocksDB-Bloom-Filter
+        double bits_per_key =
+            dsn_config_get_value_double("pegasus.server",
+                                        "rocksdb_bloom_filter_bits_per_key",
+                                        10,
```

Comment:
Why we set the bits_per_key = 10 in default? If we set it to 16, the false positive rate can be reduced more than ten times .

Reply:
Although 16 bits could provide lower false positive rate, but it will consume more space. So I decide to keep the old config by default, and this is how rocksdb do.